package.json

```diff
@@ -98,21 +107,19 @@
     "@types/lodash": "^4.14.108",
     "@types/lodash-decorators": "^4.0.0",
     "@types/node": "^8.10.17",
-    "@types/rollup": "0.51.4",
     "archiver": "^2.1.0",
     "aws-sdk": "^2.58.0",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-preset-shopify": "^16.2.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-loader": "^8.0.4",
```

Comment:
Doesn't matter too much, but `version "8.0.5"` appears in yarn.lock, so we may as well update it here.

Reply:
I'm trying to match the dependeny versions that sewing-kit uses to avoid duplication of deps. Same goes for why the rollup versions are as they are.